conftest.py

```diff
@@ -319,7 +321,7 @@ def check_float_array_equal():
     """Fixture to check if two float arrays are equal with epsilon precision tolerance."""
 
     def check_float_array_equal_impl(a, b):
-        assert numpy.all(numpy.isclose(a, b, rtol=0, atol=0.001))
+        assert check_isclose_and_shape(a, b, rtol=0, atol=0.001)
```

Comment:
would be great to add an error message here like we do for others (like check_array_equal) ! ex: `f"expected {expected}, got {got}, abs diff is {numpy.abs(got - expected)}"`